Metaverse/Source/Metaverse/Private/Components/WalletConnectTriggerComponent.cpp

```diff
@@ -145,6 +146,33 @@ void UWalletConnectTriggerComponent::Connect() {
   }
 }
 
+
+
+//
+// Disconnect
+//
```

Comment:
one line comment?

Reply:
This is not comment, it is only used for groups separation from top to bottom, instead of putting functions randomly, the purpose is for easy navigation inside a big file.
Groups are:
- Connect
- SignPersonal
- SignEip155Transaction 
- Disconnect